generate-wasm-examples/generate_wasm_examples.sh

```diff
@@ -1,26 +1,39 @@
-#!/bin/sh
+#!/bin/bash
+
+echo Generating WASM example list...
 
 # Switch to script's directory, letting it be called from any folder.
 cd $(dirname $0)
 
-./clone_bevy.sh
+# If Bevy folder already exists, pull the latest changes.
+if [[ -d bevy ]]; then
+    echo Bevy folder already exists, attempting to fetch latest changes.
+
+    cd bevy
 
-# temporary: fetch tools from main branch
-git clone --depth=1 https://github.com/bevyengine/bevy bevy-tools
+    # Attempts to fetch the latest commits, which should only happen every Bevy release.
+    git pull --depth=1
+else
+    echo Bevy folder does not exist, cloning repository.
 
-rm -rf bevy/tools
-cp -r bevy-tools/tools bevy
-rm -rf bevy-tools
-cd bevy
+    # Clone Bevy's latest branch from scratch, only downloading the latest commit.
+    git clone --depth=1 --branch=latest https://github.com/bevyengine/bevy bevy
 
-cargo run -p example-showcase -- build-website-list --content-folder content --api webgl2
-mv content ../../content/examples
+    cd bevy
+fi
 
-rm -rf content
+# Build WebGL2 example list.
+rm -rf ../../content/examples
+cargo run -p example-showcase -- build-website-list --content-folder ../../content/examples --api webgl2
```

Comment:
This line seems to be raising the error, specifically the `--content-path` argument. Going to do more testing to figure out what is going wrong...

Reply:
d1043083b546d638764f66184542636721a624a1 should fix this...